src/commonMain/resources/db/test/V72.2__AddCollaboratorsToMaps.sql

```diff
@@ -0,0 +1,17 @@
+DO
+$$
+DECLARE
+    collab1Id INTEGER;
+    collab2Id INTEGER;
+BEGIN
+    SELECT id INTO collab1Id FROM uploader WHERE "uniqueName" = 'collab1';
+    SELECT id INTO collab2Id FROM uploader WHERE "uniqueName" = 'collab2';
+
+    INSERT INTO collaboration ("mapId", "collaboratorId", "requestedAt", accepted, "uploadedAt") VALUES
+         (1, collab1Id, '2021-01-01 00:00:00', TRUE, '2021-01-01 00:00:00'),
+         (2, collab2Id, '2021-01-01 00:00:00', TRUE, '2021-01-01 00:00:00'),
+         (3, collab1Id, '2021-01-01 00:00:00', TRUE, '2021-01-01 00:00:00'),
+         (3, collab2Id, '2021-01-01 00:00:00', TRUE, '2021-01-01 00:00:00'),
+         (4, collab1Id, '2021-01-01 00:00:00', FALSE, '2021-01-01 00:00:00'),
+         (4, collab2Id, '2021-01-01 00:00:00', TRUE, '2021-01-01 00:00:00');
```

Comment:
Extra points for adding test users for sure _but_ the test maps aren't guaranteed to be maps 1-4 (although they probably will be) in staging they're actually 5-8 so this migration will probably fail :(

Reply:
I see :thinking: I was wondering if I can assume that and at the same time it didn't really matter which one they are.

How about I just select the first 4 maps instead or ? :thinking: